craco.config.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path');
+
+module.exports = {
+  reactScriptsVersion: "react-scripts" /* (default value) */,
+
+  webpack: {
+    alias: {
+      'magic-sdk' : path.resolve(__dirname, 'node_modules/magic-sdk/dist/cjs/index.js')
+    }
+  },
+};
```

Comment:
@Adamj1232 not a _huge_ deal but maybe add a line break here?

Reply:
Ahh yes, good call! Prettier-ified the whole file and it looks a lot better!